moto/ec2/models/instances.py

```diff
@@ -550,6 +550,8 @@ def get_instance(self, instance_id):
     def add_instances(self, image_id, count, user_data, security_group_names, **kwargs):
         location_type = "availability-zone" if kwargs.get("placement") else "region"
         default_region = "us-east-1"
+        describe_key_pairs = self.describe_key_pairs
+        describe_key_pairs(key_names=[kwargs.get("key_name")])
```

Comment:
```suggestion
        self.describe_key_pairs(key_names=[kwargs.get("key_name")])
```
Minor nitpick - I don't think reassigning the method does anything for readability